graphql_compiler/__init__.py

```diff
@@ -17,11 +17,20 @@
     DIRECTIVES, EXTENDED_META_FIELD_DEFINITIONS, GraphQLDate, GraphQLDateTime, GraphQLDecimal,
     insert_meta_fields_into_existing_schema, is_meta_field
 )
+from .schema_generation.schema_graph import SchemaGraph
+from .schema_generation.schema_properties import ORIENTDB_BASE_VERTEX_CLASS_NAME
+from .schema_generation.graphql_schema import get_graphql_schema_from_schema_graph
+from .schema_generation.utils import toposort_classes
 
 
 __package_name__ = 'graphql-compiler'
 __version__ = '1.10.0'
 
+# Match query used to generate OrientDB records that are themselves used to generate GraphQL schema.
+ORIENTDB_SCHEMA_RECORDS_QUERY = 'SELECT FROM (SELECT expand(classes) FROM metadata:schema) '\
```

Comment:
Use parens instead of `\`:

```
ORIENTDB_SCHEMA_RECORDS_QUERY = (
    'foo'
    'bar'
)
```

But also, this should probably not be in the root `__init__.py` file, since it's very much an implementation detail of the schema generation.